conda/recipes/versions.yaml

```diff
@@ -39,7 +39,7 @@ clang_version:
 cmake_version:
   - '>=3.20.1,<3.21'
 cmake_format_version:
```

Comment:
Should this be renamed to `cmakelang_version`?

Reply:
Yeah I guess may as well. Done.